qa/standalone/ceph-helpers.sh

```diff
@@ -152,11 +152,15 @@ function test_setup() {
 #
 function teardown() {
     local dir=$1
+    local dumplogs=$2
     kill_daemons $dir KILL
     if [ `uname` != FreeBSD ] \
         && [ $(stat -f -c '%T' .) == "btrfs" ]; then
         __teardown_btrfs $dir
     fi
+    if [ "$dumplogs" = "1" ]; then
+        display_logs $dir
+    fi
```

Comment:
@dzafman 
Yup, that is exactly the reason I haven't worked on it. I have no clue as to teuthology detection.
In my local jenkins, I could move stuff to `/tmp/error.$$/.....`